<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveGraphWindow</class>
 <widget class="QMainWindow" name="SaveGraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>save graph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/skate.jpg</normaloff>:/skate.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bauhaus 93</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SAVE GRAPH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>graph name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>graph_name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="pdfCheckBox">
      <property name="text">
       <string>save as PDF flie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="pngCheckBox">
      <property name="text">
       <string>save as PNG file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bmpCheckBox">
      <property name="text">
       <string>save as BMP file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="jpgCheckBox">
      <property name="text">
       <string>save as JPG file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveGraphButton">
      <property name="text">
       <string>save graph</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
